cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/AddCommand.java

```diff
@@ -51,7 +51,7 @@
 /**
  * New command for adding a new module.
  */
-@CommandLine.Command(name = ADD_COMMAND, description = "Add a new module to Ballerina project")
+@CommandLine.Command(name = ADD_COMMAND, description = "Adds a new module to a Ballerina project.")
```

Comment:
This should be "Add a new module to an existing Ballerina project." 

It is not "Adds" IMO. Please refer other command-line tools

Reply:
Ack. +1 Thanks.